### ISSUE TITLE
Module cmp-enhancer only supports java versions up to 1.6

### ISSUE THREAD

**Issue by svendiedrichsen** [payara/Payara#3756]
# Description #
----------

The module cmp-enhancer only supports java versions up to 1.6. If Payara does not support java versions <= 1.6 anymore this module seems to be obsolete or it needs to be updated to support java versions > 1.6.

`com.sun.jdo.api.persistence.enhancer.classfile.ClassFile`
```
...
  public static final short[][] jdkMajorMinorVersions = new short[][]{
    new short[]{45, 3}, // jdk 1.1
    new short[]{46, 0}, // jdk 1.2
    new short[]{47, 0}, // jdk 1.3
    new short[]{48, 0}, // jdk 1.4
    new short[]{49, 0}, // jdk 1.5
    new short[]{50, 0}  // jdk 1.6
  };
...
```

**Comment by smillidge** [comment]
JDO is old but was pretty cool.

**Comment by svendiedrichsen** [comment]
Agreed. But it is unusable anymore in the state it is in. Should this be removed or updated then.

**Comment by Cousjava** [comment]
I have made PR #3764 to add support for later JDKs.

**Comment by svendiedrichsen** [comment]
I wonder if this list is really the only thing that is missing to support jdks > 1.6.

**Comment by smillidge** [comment]
Fixed by #3764
